model/src/main/java/uk/gov/hmcts/fees2/register/data/service/impl/FeeServiceImpl.java

```diff
@@ -171,7 +171,7 @@ public FeeLookupResponseDto lookup(LookupFeeDto dto) {
 
         defaults(dto);
 
-        List<Fee> fees = search(dto);
+        List<Fee> fees = search(dto).stream().filter(fee -> fee.getCurrentVersion(true) != null).collect(Collectors.toList());
```

Comment:
Is it easy to push filter to db query level? 
btw, can you cleanup of these lines in that method.. no significant at all. https://github.com/hmcts/ccfr-fees-register-app/blob/8e74b6ae95a1f4fbd48548588583673f72a4e5e5/model/src/main/java/uk/gov/hmcts/fees2/register/data/service/impl/FeeServiceImpl.java#L186-L194